react/features/base/tracks/middleware.any.ts

```diff
@@ -203,7 +202,7 @@ function _setMuted(store: IStore, { ensureTrack, muted }: {
             setTrackMuted(jitsiTrack, muted, state, dispatch)
                 .catch(() => dispatch(trackMuteUnmuteFailed(localTrack, muted)));
         }
-    } else if (!muted && ensureTrack && (typeof APP === 'undefined' || isPrejoinPageVisible(state))) {
+    } else if (!muted && ensureTrack) {
         typeof APP !== 'undefined' && dispatch(gumPending([ mediaType ], IGUMPendingState.PENDING_UNMUTE));
```

Comment:
Why do you need to remove these?

Reply:
I pushed a commit to fix the issue of duplicate tracks.